src/include/simeng/OS/SyscallHandler.hh

```diff
@@ -428,7 +436,7 @@ class SyscallHandler {
   SyscallInfo currentInfo_ = {};
 
   /** Path to the root of the replacement special files. */
-  const std::string specialFilesDir_ = SIMENG_BUILD_DIR "/specialFiles";
+  // const std::string specialFilesDir_ = SIMENG_BUILD_DIR "/specialFiles";
```

Comment:
Isn't this meant to be removed rather than just commented out?

Reply:
Yes it was